src/main/resources/scripts/kill counter.sk

```diff
@@ -2,25 +2,24 @@
 # Counts kills per life, in total, and the highest kill streak.
 #
 
-# Defaults are useful here, as a player's kills would be '<none>' if he hasn't killed anything yet,
-# but with defaults defined those will be used in that case.
-variables:
-	{kill counter.%player%.kills_session} = 0
```

Comment:
This introduces a potentially useful feature. The fact that docs do not explicitly mention it... Well, not an issue in example scripts. Docs should be fixed, not this.

Reply:
so you want me to add the variables section back?